app/controllers/navigation/Navigator.scala

```diff
@@ -165,13 +163,13 @@ class Navigator @Inject() (
   }
 
   val standardCacheItemDependent: PartialFunction[DeclarationPage, (ExportsDeclaration, Mode, String) => Call] = {
-    case AdditionalDocumentsRequired if waiver999LConfig.is999LEnabled => additionalDocumentsSummaryPreviousPage
-    case AdditionalDocumentsSummary if waiver999LConfig.is999LEnabled  => additionalDocumentsSummaryPreviousPage
-    case AdditionalDocument if waiver999LConfig.is999LEnabled          => additionalDocumentsPreviousPage
-    case AdditionalDocumentsRequired                                   => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocumentsSummary                                    => additionalDocumentsSummaryNoWaiverPreviousPage
-    case AdditionalDocument                                            => additionalDocumentsNoWaiverPreviousPage
-    case NactCodeFirst                                                 => nactCodeFirstPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryPreviousPage
+    case AdditionalDocument          => additionalDocumentsPreviousPage
+    case AdditionalDocumentsRequired => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocumentsSummary  => additionalDocumentsSummaryNoWaiverPreviousPage
+    case AdditionalDocument          => additionalDocumentsNoWaiverPreviousPage
```

Comment:
Why were these 3 lines not removed?
The compiler is also complaining (with just a warning btw) about them.